bin/install-cassandra.sh

```diff
@@ -0,0 +1,46 @@
+SAVEDIR=$PWD
+# Change this ROOR_DIR to your preferred instalation directory, and make sure to update cassandraRootDir at SSTableTest.java
+ROOT_DIR="/tmp/"
+
+[ -e $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz ] && rm $ROOT_DIR/apache-cassandra-3.11.6-bin.tar.gz
```

Comment:
why delete it if it exists? Perhaps you should do just the opposite, only download it if it does not exist.
Maybe you can make 3.11.6 a CASSANDRA_VERSION variable.